src/document/DocumentAnnotator.scss

```diff
@@ -11,15 +11,41 @@
         pointer-events: none;
     }
 
-    &.ba-is-highlighting {
+    &.ba-discoverability-enabled {
         .textLayer {
-            > span {
-                @include ba-HighlightCreator-cursor;
+            pointer-events: none; // Ensures that click events are received on the create region layer
+
+            span {
+                pointer-events: auto; // Text layer still receives text highlight events
+            }
+        }
+
+        &.ba-is-create--highlight {
+            .textLayer {
+                > span {
+                    @include ba-HighlightCreator-cursor;
+                }
+
+                ::selection {
+                    background-color: rgba($bdl-yellow, .33);
+                }
             }
+        }
 
-            ::selection {
-                background-color: rgba($bdl-yellow, .33);
+        &.ba-is-create--region {
+            .textLayer > span {
+                pointer-events: none;
```

Comment:
This logic of turning off `pointer-events` on the `.textLayer > span` will need to move into preview SDK per this discussion: https://github.com/box/box-annotations/pull/588#discussion_r488150955

Reply:
Would this PR not be able to be merged until that change is completed first?

Reply:
No it should be unrelated since no one is using the discoverability flip yet